src/pymedphys/gamma/_level2/gammashell.py

```diff
@@ -399,14 +399,14 @@ def calculate_min_dose_difference(options, distance, to_be_checked,
 
     num_points_in_shell = np.shape(coordinates_at_distance_shell)[1]
 
-    estimated_ram_needed = (np.uint64(num_points_in_shell)
-                            * np.sum(to_be_checked).astype(np.uint64)
-                            * np.uint64(32)
-                            * np.uint64(num_dimensions)
+    estimated_ram_needed = (num_points_in_shell
+                            * np.sum(to_be_checked)
+                            * 32
+                            * num_dimensions
```

Comment:
These `np.uint64` were written in to fix the following bug:

https://gitlab.com/pymedphys/pymedphys/issues/17

I couldn't ever replicate the issues on my machines so I'm not sure how to verify that removing them doesn't reintroduce the bug.

I have a few ideas here though.

Reply:
I know.

There's no need to explicitly cast all of the multiplied together variables to uint64 though. One cast is sufficient.

Reply:
Maybe we'll just remove these changes from the merge. I don't think we lose anything remotely noticeable from explicitly casting each of them individually.

I'd mistakenly thought one of the variables was a largish array, but they're all either scalars are single-valued arrays, right?

Reply:
Maybe we'll just remove these changes from the merge. I don't think we
 lose anything remotely noticeable from explicitly casting each of them
 individually.

 I'd mistakenly thought one of the variables was a largish array, but
 they're all either scalars are single-valued arrays, right?